sdk/monitor/monitor-opentelemetry-exporter/src/generated/applicationInsightsClient.ts

```diff
@@ -45,7 +45,7 @@ export class ApplicationInsightsClient extends coreClient.ServiceClient {
       userAgentOptions: {
         userAgentPrefix
       },
-      baseUri: options.endpoint ?? options.baseUri ?? "{Host}/v2.1"
+      endpoint: options.endpoint ?? options.baseUri ?? "{Host}/v2.1"
```

Comment:
Changes to autogenerated files would be temporary, we need to update JSON definition file used by swagger